packages/ui/components/top-banner/TopBanner.tsx

```diff
@@ -23,17 +25,20 @@ export function TopBanner(props: TopBannerProps) {
     <div
       data-testid="banner"
       className={classNames(
-        "flex min-h-[40px] w-full items-start justify-between gap-8 bg-gray-50 py-2 px-4 text-center lg:items-center",
+        "alert-banner flex min-h-[40px] w-full items-start justify-between gap-8 bg-gray-50 py-2 px-4 text-center lg:items-center [&+.alert-banner]:border-t [&+.alert-banner]:border-black",
```

Comment:
Added a divider line between to banners in the edge case where 2 banners are visible at the same time.

Reply:
but but `divide-y`

Reply:
HAHAHA. This guy. You're totally right. It's my way too old brain trying to use old skool CSS where the new kid on the block (tailwind) does it way better. I'll change this indeed, great suggestion 😁

Reply:
Fixed!